src/main/java/minijava/ir/optimize/ConstantControlFlowOptimizer.java

```diff
@@ -14,7 +18,7 @@
  * <p>The {@link Proj} node that is no longer relevant is replaced with a {@link Bad} node. A
  * subsequent run of an {@link Optimizer} that removes such nodes is required.
  */
```

Comment:
Yeah, I'll rather pull it out into a different class. I have absolutely no clue how to name it, though...